internal/service/imagebuilder/image_recipe_data_source.go

```diff
@@ -43,11 +44,11 @@ func dataSourceImageRecipe() *schema.Resource {
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
 									names.AttrDeleteOnTermination: {
-										Type:     schema.TypeBool,
+										Type:     nullable.TypeNullableBool,
```

Comment:
I definitely don't understand why we need an entire package for a single type. 😖 😕 Feels like Java 🤮